third_party/canvaskit/BUILD.gn

```diff
@@ -20,9 +26,15 @@ group("canvaskit_group") {
 # This toolchain is only to be used by canvaskit_chromium_group below.
 wasm_toolchain("canvaskit_chromium") {
   extra_toolchain_args = {
+    # Remove ICU data.
     skia_use_icu = false
     skia_use_client_icu = true
     skia_icu_bidi_third_party_dir = "//flutter/third_party/canvaskit/icu_bidi"
+
+    # Remove image codecs.
```

Comment:
I would instead explain that in Chromium we use ImageDecoder instead of the wasm codecs.